src/components/clientDashboard/enrollments/ClientAssessments.tsx

```diff
@@ -45,6 +49,10 @@ const ClientAssessments = () => {
     [clientId]
   );
 
+  const filters = useFilters({
+    type: getInputTypeForRecordType('Assessment'),
```

Comment:
Let me know how you think this looks - I did also consider moving the `getInputTypeForRecordType` inside of `useFilters` so that it's more of a default, but the caller then has to pass in `recordType` anyway. I ended up thinking this was the cleanest approach, but I'd be glad for cleaner suggestion.

Reply:
Hmmmm I see. Honestly I think just specifying `'AssessmentFilterOptions'` feels more straightforward as a pattern. Either way is fine imo.
It would be nice if we had a generated enum for those strings,  but we don't currently.